core/src/main/java/org/jdbi/v3/BasicHandle.java

```diff
@@ -244,7 +249,8 @@ public Update createStatement(String sql)
                           sql,
                           new ConcreteStatementContext(globalStatementAttributes, MappingRegistry.copyOf(mappingRegistry)),
                           timingCollector,
-                          foreman);
+                          foreman,
+                          containerFactoryRegistry);
```

Comment:
Can we revert this constructor argument change for classes that don't implement ResultBearing?